bin/demeteorizer

```diff
@@ -42,7 +48,9 @@ var options = {
   input: process.cwd(),
   directory: Program.output,
   json: Program.json,
-  architecture: Program.architecture
+  architecture: Program.architecture,
+  verbose: Boolean(Program.verbose),
+  server: Program.server || "localhost:3000"
```

Comment:
Ah I read the documentation wrong. 😢 . But you can remove [this line](https://github.com/jwall149/demeteorizer/blob/c2989cbfe402525d203d38dff35c0017a4570403/lib/build.js#L7) since you're setting a default elsewhere.